### PR TITLE
test: how to run a single .t file

### PR DESCRIPTION
**Pull request by harigowtham**
Signed-off-by: hari gowtham <hgowtham@redhat.com>

### CODE REVIEW

**Comment by amarts** [comment]
We have step to run the single step in https://github.com/gluster/glusterfs README, but this is good.

File: docs/Developer-guide/Development-Workflow.md [hunk 1]
```diff
@@ -344,16 +344,22 @@ code change is actually verified by the test case.
...
+To run all regression tests locally, run below script from glusterfs root directory.
 
 ```console
 # ./run-tests.sh
 ```
 
+To run a single regression test locally, run the below command.
+
+```console
+# prove <path_to_the_file>
```
Comment (BarakSason) [inline docs/Developer-guide/Development-Workflow.md:360]: Maybe # prove -vf <path_to_the_file> as it provides detailed information?

Comment (harigowtham) [reply docs/Developer-guide/Development-Workflow.md:360]: -vf being options I wasn't sure what are the suggested ones to be used. Like I'm used to using -r as well. though it isn't needed for a single test, it comes handy to use the same command in other situations as well. Anyways for now, I'll update for these two options alone. let me know if anything has to be added. Thanks @BarakSason

File: docs/Developer-guide/Development-Workflow.md [hunk 2]
```diff
@@ -344,16 +344,22 @@ code change is actually verified by the test case.
 Regression tests (i.e, execution of all test cases accumulated with
 every commit) is not automatically triggered as the test cases can be
 extensive and is quite expensive to execute for every change submission
-in the review/resubmit cycle. Instead it is triggered by the
-maintainers, after code review. Passing the regression test is a
+in the review/resubmit cycle. Instead it is triggered when the patch
+contributor issues a +1 on gerrit. Passing the regression test is a
```
Comment (sunnyku) [inline docs/Developer-guide/Development-Workflow.md:348]: Code or Verified (+1)

Comment (kshithijiyer) [reply docs/Developer-guide/Development-Workflow.md:348]: I think this should be `Verified: +1`.

Comment (harigowtham) [reply docs/Developer-guide/Development-Workflow.md:348]: done

Comment (harigowtham) [reply docs/Developer-guide/Development-Workflow.md]: done

File: docs/Developer-guide/Development-Workflow.md [hunk 3]
```diff
@@ -344,16 +344,24 @@ code change is actually verified by the test case.
...
 # ./run-tests.sh
 ```
 
+To run a single regression test locally, run the below command.
+
+```console
+# prove -vf <path_to_the_file>
+```
+
+NOTE: The testing framework needs perl-Test-Harness package to be installed.
```
Comment (kshithijiyer) [inline docs/Developer-guide/Development-Workflow.md:363]: Would be nice if we can make NOTE bold using something like **NOTE**.

Comment (harigowtham) [reply docs/Developer-guide/Development-Workflow.md:363]: done

**Pull request merged**